src/App.js

```diff
@@ -1,10 +1,21 @@
 import React from 'react';
 import './App.css';
+import SearchBar from './Components/SearchBar';
+import SearchResults from './Components/SearchResults';
+import Playlist from './Components/Playlist';
+
 
 function App() {
   return (
-    <div className="App">
-
+    <div>
```

Comment:
If my div that wraps around the entire component isn’t styled, I like to replace them with an empty tag <> & </>